<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateSettingsDialog</class>
 <widget class="QDialog" name="UpdateSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="updateSettingsGroupBox">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>298</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QCheckBox" name="createNewItemsCheck">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>65</y>
      <width>228</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Create New Item(s) For Unassigned Sprites</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="reloadItemAttributesCheck">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>42</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload Item Attributes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="generateSignatureCheck">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>88</y>
      <width>187</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Image Signatures (Slow)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reassignUnmatchedSpritesCheck">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>19</y>
      <width>216</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reassign Items With Unmatched Sprites</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>149</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
